addons/account/models/account_journal.py

```diff
@@ -34,7 +34,7 @@ class AccountJournal(models.Model):
     _description = "Journal"
     _order = 'sequence, type, code'
     _inherit = ['portal.mixin',
-                'mail.alias.mixin',
+                'mail.alias.mixin.light',
```

Comment:
If we end up keeping the light mixin, I would merge this commit with the preceding one, to avoid having stuff re-writing itself too much.

Reply:
That's the spirit yes, I keep a separate commit as the "optional" mixin was just something I wanted to try.